spec/swagger_helper.rb

```diff
@@ -32,7 +32,22 @@
       #     scopes: {}
       #   }
       # },
+      # rubocop:disable Style/NumericLiterals
       definitions: {
+        payment: {
+          type: :object,
+          properties: {
+            agency_id: { type: :string, example: '3fa57706-f5bb-4d40-9350-85871f698d52' },
+            amount: { type: :integer, minimum: 0, example: 19999 },
```

Comment:
Rather than disabling rubocop can we use the underscore style for the larger numbers?

```suggestion
            amount: { type: :integer, minimum: 0, example: 19_999 },
```

Reply:
We definitely can.  
I'm not a fan of this style (hence my resistance). :-)
(It's also representing 'One-hundred, nineteen dollars and ninety-nine cents'  in my head.)

However:  this is your project and you get to set the styles.  :-)  
So I'll make the change.

Reply:
Definitely get it, thanks for being flexible! :)